package.json

```diff
@@ -12,69 +12,60 @@
     "node": ">= 6"
   },
   "scripts": {
-    "lint": "eslint index.js lib test",
-    "pretest": "npm run lint",
+    "commitlint": "commitlint --from=master",
+    "lint": "eslint --cache lib test",
+    "release": "standard-version",
+    "security": "npm audit",
     "test:only": "jest",
-    "test:coverage": "jest --coverage",
-    "test": "jest",
-    "release": "standard-version"
+    "test:coverage": "npm run test:only -- --coverage",
+    "test:watch": "npm run test:only --watch",
```

Comment:
/cc @hiroppy watch should be without coverage, so better use `test:only` (we need fix it in webpack-dev-server)

Reply:
> watch should be without coverage

hm... honestly, I want to be able to see coverage when running as `test:watch`.